lib/kvao/index.js

```diff
@@ -0,0 +1,14 @@
+'use strict';
+
+function KeyValueAccessObject() {
+};
+
+module.exports = KeyValueAccessObject;
+
+KeyValueAccessObject.get = require('./get');
+KeyValueAccessObject.set = require('./set');
+
```

Comment:
To avoid the situation we have in `lib/dao.js`, where the file contains 3000 lines of code and still grows.

By splitting the methods into separate files, the project is also easier navigate around, at least in my experience/opinion.